state_of_css.yml

```
@@ -495,78 +799,336 @@ translations:
   - key: opinions.selector_nesting_to_be_avoided.title
-    t: Вкладені селектори
+    t: Вкладення селекторів
 
   - key: opinions.css_is_programming_language
-    t: CSS - це мова програмування
+    t: CSS — це мова програмування
   - key: opinions.css_is_programming_language.title
     t: Мова програмування
 
   - key: opinions.enjoy_writing_css
     t: Мені подобається писати CSS
   - key: opinions.enjoy_writing_css.title
-    t: Задоволення
+    t: Насолода
 
-  - key: opinions_others.css_missing_features.others
-    t: Чого вам не вистачає в CSS?
-  - key: opinions_others.css_missing_features.others.description
-    t: Можливості, які ви хотіли б бачити в CSS.
-  - key: opinions_others.css_missing_features.others.note
+  # Browser interoperability question
+  - key: opinions_others.browser_interoperability_features.others
+    t: Несумісність між браузерами
+  - key: opinions_others.browser_interoperability_features.others.description
     t: >
-      Ці дані були нормалізовані на основі відповідей у вільній формі.
-      Подивитися оригінальні відповіді можна на сайті [whatsmissingfromcss.com](http://whatsmissingfromcss.com/).
+      Якими наявними функціями CSS ви не можете користуватися (або взагалі уникаєте)
+      через відсутність підтримування або відмінності між браузерами?
+
+  - key: usage.css_interoperability_features
+    aliasFor: opinions_others.browser_interoperability_features.others
+  - key: usage.css_interoperability_features.question
+    aliasFor: opinions_others.browser_interoperability_features.others.description
+
+  - key: usage.css_interoperability_features_freeform
+    aliasFor: opinions_others.browser_interoperability_features.others
+  - key: usage.interoperability_features_freeform
+    aliasFor: opinions_others.browser_interoperability_features.others
+  - key: usage.interoperability_features_freeform.others
+    aliasFor: opinions_others.browser_interoperability_features.others
+
+  - key: usage.interoperability_features_freeform.description
+    aliasFor: opinions_others.browser_interoperability_features.others.description
+  - key: usage.interoperability_features_freeform.others.description
+    aliasFor: opinions_others.browser_interoperability_features.others.description
+
+  - key: options.css_interoperability_features.math_functions
+    aliasFor: features.math_functions
+  - key: options.css_interoperability_features.css_nesting
+    aliasFor: features.css_nesting
+  - key: options.css_interoperability_features.at_container
+    aliasFor: features.at_container
+
+  - key: options.css_interoperability_features.color_functions
+    aliasFor: features.color_functions
+  - key: options.css_interoperability_features.color_functions.description
+    aliasFor: features.color_functions.description
+  - key: options.css_interoperability_features.cascade_layers
+    aliasFor: features.cascade_layers
+  - key: options.css_interoperability_features.scrollbar
+    t: Стилі для елементам гортання
```

```suggestion
    t: Стилі для смуги гортання
```
У нас "смуга прокручення"